Common/Constants/Constants.cs

```diff
@@ -9,5 +9,6 @@ public class Constants
         public const int Validity = 1;
         public const string PasswordInvalidMessage = "The Password field is not a valid password";
         public const string PasswordRegularExpression = @"^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&+=.,\-_!])([a-zA-Z0-9 @#$%^&+=*.,\-_!]){8,}$";
+        public const string TokenPartTest = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9";

```

Comment:
in this particular situation, this token is not 'common' resource, it is specific to one unit test. So it is better to store it in unit test